script.js

```diff
@@ -11,15 +17,121 @@ toggleMenu.addEventListener("click", (e) => {
     menuOpen = false;
   }
 
-  navlinks.classList.toggle('open');
+  navlinks.classList.toggle("open");
 });
 
-
-
 function closeMenu() {
-  toggleMenu.classList.remove('open');
-  navLinks.classList.remove('open');
+  toggleMenu.classList.remove("open");
+  navlinks.classList.remove("open");
   menuOpen = false;
 }

```

Comment:
[OPTIONAL] Good Job, but it would be best to use ES6 arrow function. By using this, it shows you are updated and it would even help you in React.